PhysicsTools/NanoAOD/python/boostedTaus_cff.py

```diff
@@ -11,7 +11,8 @@
 
 finalBoostedTaus = cms.EDFilter("PATTauRefSelector",
     src = cms.InputTag("slimmedTausBoosted"),
-    cut = cms.string("pt > 40 && tauID('decayModeFindingNewDMs') && (tauID('byVVLooseIsolationMVArun2DBoldDMwLT') || tauID('byVVLooseIsolationMVArun2DBnewDMwLT'))")
+    cut = cms.string("pt > 40 && tauID('decayModeFindingNewDMs') && (tauID('byVVLooseIsolationMVArun2DBoldDMwLT') || tauID('byVVLooseIsolationMVArun2DBnewDMwLT'))"),
+    lazyzEval = cms.untracked.bool(True)                            
```

Comment:
typo: `lazyzEval` -> `lazyEval`
But actually is this implemented for `PATTauRefSelector`? I don't seem to find it anywhere...

Reply:
fixed. Thanks!